sqlglot/expressions.py

```diff
@@ -1479,6 +1479,123 @@ class Insert(Expression):
         "alternative": False,
     }
 
+    def insert(
```

Comment:
this method is a bit strange, i'm not sure we need it, why would you ever use this and not just exp.insert?